molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/MultiAllelicResultFilter.java

```diff
@@ -0,0 +1,72 @@
+package org.molgenis.data.annotation.entity.impl;
+
+import com.google.common.collect.FluentIterable;
+import org.molgenis.data.AttributeMetaData;
+import org.molgenis.data.Entity;
+import org.molgenis.data.annotation.entity.ResultFilter;
+import org.molgenis.data.vcf.VcfRepository;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+//TODO before merging with the master this should be unit-tested
+public class MultiAllelicResultFilter implements ResultFilter
+{
+
+	private List<AttributeMetaData> attributes;
+	public MultiAllelicResultFilter(List<AttributeMetaData> attributes)
+	{
+		this.attributes = attributes;
+	}
+
+	@Override
+	public Collection<AttributeMetaData> getRequiredAttributes()
+	{
+		return Arrays.asList(VcfRepository.REF_META, VcfRepository.ALT_META);
+	}
+
+	@Override
+	public com.google.common.base.Optional<Entity> filterResults(Iterable<Entity> results, Entity annotatedEntity)
```

Comment:
make this a correct import (short key is mac shift m when the cursor is before Optional)